asserts/system_user.go

```diff
@@ -230,6 +236,17 @@ func assembleSystemUser(assert assertionBase) (Assertion, error) {
 	if err != nil {
 		return nil, err
 	}
+	serials, err := checkStringList(assert.headers, "serials")
+	if err != nil {
+		return nil, err
+	}
+	if len(serials) > 0 && assert.Format() < 1 {
+		return nil, fmt.Errorf(`the "serials" header is only available for format 1 or greater`)
+	}
+	if len(serials) > 0 && len(models) != 1 {
+		return nil, fmt.Errorf(`the "serials" header must specify exactly one model`)
```

Comment:
'in the presence of the "serials" header "models" should ...' ?

Reply:
Updated with "must" instead of "should" as this is a hard requirement (but happy to change if you feel strongly about the "should")

Reply:
no, must is fine